framework/test/mocha/common/lisk-mocha-runner/index.js

```diff
@@ -144,6 +144,20 @@ if (testPathPattern && testPathPattern.indexOf('-') === 0) {
 mochaCliOptions.unshift('--', '--opts', 'framework/test/mocha/mocha.opts');
 
 // Execute lisk mocha runner
-executeTests(testType, testPathPattern, mochaCliOptions).catch(error =>
-	console.error(error.message)
-);
+executeTests(testType, testPathPattern, mochaCliOptions)
+	.then(() => {
+		if (
+			Object.keys(state.tests.failed).length ||
+			Object.keys(state.tests.killed).length
+		) {
+			console.error(
+				'\n\nFAILED: Some tests were not passed. Test runner failed!'
```

Comment:
Instead of showing some tests were not passed, maybe some number? or it's shown already?

Reply:
Yes in the summary section we are getting some numbers and messages were updated.